docs/src/examples/basic.jl

```diff
@@ -0,0 +1,43 @@
+# # A basic example
```

Comment:
Where do we document the licenses for these files now?

Reply:
I re-added them with a `#src` tag.